src/MapzenSearch.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  MapzenSearch.swift
+//  Pods
+//
+//  Created by Sarah Lensing on 3/27/17.
+//
+//
+
+import Pelias
+
+public class MapzenSearch : NSObject {
+
+  public static let sharedInstance = MapzenSearch()
+  private let peliasSearchManager = PeliasSearchManager.sharedInstance
+
+  //In seconds
+  public var autocompleteTimeDelay: Double {
+    get {
+      return peliasSearchManager.autocompleteTimeDelay
+    }
+    set(delay) {
+      peliasSearchManager.autocompleteTimeDelay = delay
+    }
+  }
+
+  public var baseUrl: URL {
+    get {
+      return peliasSearchManager.baseUrl
+    }
+    set(url) {
+      peliasSearchManager.baseUrl = url
+    }
+  }
+
+  public var urlQueryItems: [URLQueryItem]? {
+    get {
+      return peliasSearchManager.urlQueryItems
+    }
+    set(queryItems) {
+      peliasSearchManager.urlQueryItems = queryItems
+    }
+  }
+
+  fileprivate override init() {
+  }
+
+  public func search(_ config: SearchConfig) -> SearchOperation {
+    let peliasOperation = peliasSearchManager.performSearch(config.peliasConfig);
+    return SearchOperation.init(peliasOperation)
+  }
+
+  public func reverseGeocode(_ config: ReverseConfig) -> SearchOperation {
+    let peliasOperation = peliasSearchManager.reverseGeocode(config.peliasConfig)
+    return SearchOperation.init(peliasOperation)
+  }
+
+  public func autocompleteQuery(_ config: AutocompleteConfig) -> SearchOperation {
+    let peliasOperation = peliasSearchManager.autocompleteQuery(config.peliasConfig)
+    return SearchOperation.init(peliasOperation)
+  }
+
+  public func placeQuery(_ config: PlaceConfig) -> SearchOperation {
+    let peliasOperation = peliasSearchManager.placeQuery(config.peliasConfig)
+    return SearchOperation.init(peliasOperation)
+  }
+
+  public func cancelOperations() {
+    peliasSearchManager.cancelOperations()
+  }
+}
+
+@objc(MZSearchOperation)
+public class SearchOperation : Operation {
+
+  let peliasOperation: PeliasOperation
+
+  init(_ op: PeliasOperation) {
+    peliasOperation = op
+  }
+
+  override public func start() {
```

Comment:
I wrapped it because I didnt want to recreate a new operation given the `PeliasSearchOperation` that is returned by the `PeliasSearchManager`. It seemed more complicated (I'd have to add it to the internal queue and remove the returned operation from that queue).